provision-osx/tasks/configure_buildfarm_proxy.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+-
+  name: Set node proxy configuration with authentication
+  shell: "networksetup -{{item}} {{proxy_device}} {{proxy_url}} {{proxy_port}} on {{proxy_user}} {{proxy_pass}}"
```

Comment:
I don't think it affects the use of the code but we have been following the syntax {{ variable }} instead of {{variable}}